Bottom: 3b0160d63fee720dccedd8e634ca47dc03f2ca6b
Top:    460d51317822467d5f0923d7bd28b66882506758
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-29 15:39:34 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
